src/NuGet.Indexing/VersionResult.cs

```diff
@@ -10,21 +10,37 @@ public class VersionResult
     {
         public VersionResult()
         {
-            VersionDetails = new List<VersionDetail>();
+            AllVersionDetails = new List<VersionDetail>();
         }
 
-        public List<VersionDetail> VersionDetails { get; private set; }
+        public List<VersionDetail> AllVersionDetails { get; private set; }
 
-        public IEnumerable<string> GetVersions(bool onlyListed)
+        public IEnumerable<VersionDetail> SemVer2VersionDetails { get { return AllVersionDetails; } }
```

Comment:
nit: you can use
`=> AllVersionDetails;`
instead of 
`{ get { return AllVersionDetails; } }`